### PR TITLE
ola 0.9.5

### PR DESCRIPTION
**Pull request by brunchboy**
Update the version of the Open Lighting Archicitecture to
the latest stable release, currently 0.9.5.

### CODE REVIEW

File: Library/Formula/ola.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 class Ola < Formula
   homepage "http://www.openlighting.org/ola/"
-  url "https://github.com/OpenLightingProject/ola/releases/download/0.9.3/ola-0.9.3.tar.gz"
-  sha1 "f6a81087761218063a4bb8006b73ffa407cd0170"
+  url "https://github.com/OpenLightingProject/ola/releases/download/0.9.5/ola-0.9.5.tar.gz"
+  sha1 "e9b0d8e4e3e4acd696cd7a0877d72d593727a7bc"
```
Comment (DomT4) [inline Library/Formula/ola.rb]: ` sha256 ` would be cool, if you're feeling particularly kind :smiley_cat:

Comment (brunchboy) [inline Library/Formula/ola.rb]: I would be happy to oblige if you could explain what you mean?

Comment (DomT4) [inline Library/Formula/ola.rb]: Oh, just change the ` sha1 ` to a ` sha256 ` checksum. It's not a huge thing if you don't want to push another commit. In this case the ` sha256 ` is ` "7c24ac98e865b4c354a04563b88012e782205ffd932a05cf944273f6f5ea82ca" `.

Comment (brunchboy) [inline Library/Formula/ola.rb]: That is no problem! I can do that, I just was unaware how to calculate it (I did some research and came up with the same value you did.)

So, in addition to changing the value, I should change the key "sha1" to be "sha256", I expect?

Comment (DomT4) [inline Library/Formula/ola.rb]: You can either do ` brew fetch /path/to/formula.rb -fs ` which will output the ` sha1 ` and ` sha256 ` or you can download it manually and then ` openssl dgst -sha256 /path/to/file.tar.gz `.

> So, in addition to changing the value, I should change the key "sha1" to be "sha256", I expect?

Yup! Thanks :heart:

**Comment by xu-cheng** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

**Pull request closed without merging**